src/configure

```diff
@@ -4582,30 +4577,6 @@ then :
 fi
 
 
-if test "x$with_magma" = "xyes"; then :
-
-$as_echo "#define HYPRE_USING_MAGMA 1" >>confdefs.h
-
-fi
-
-
-# Check whether --with-magma-include was given.
-if test "${with_magma_include+set}" = set; then :
-  withval=$with_magma_include; for dmagma_inc_dir in $withval; do
-    MAGMA_INCLUDE="-I$magma_inc_dir $MAGMA_INCLUDE"
- done
-
-fi
-
-
-
-# Check whether --with-magma-lib was given.
-if test "${with_magma_lib+set}" = set; then :
-  withval=$with_magma_lib; for magma_lib in $withval; do
-    MAGMA_LIBS="$MAGMA_LIBS $magma_lib"
- done
-
-fi
```

Comment:
@victorapm , when I reran `config/update.sh`, there were some deletions of magma-related stuff. This stuff mostly looks redundant, but I wanted to double check with you that this looks OK.

Reply:
Ah, these deletions actually break the build. I get the error:

icpx: error #10005: unable to open argument file 'MAGMA_LIBS@'

I notice that we have both `MAGMA_LIBS` and `HYPRE_MAGMA_LIB` listed in Makefile.config.in:

https://github.com/hypre-space/hypre/blob/master/src/config/Makefile.config.in#L177

Should one of these sets of magma-related variables be removed?

Reply:
Hi Wayne, yes, the ones without the hypre lrefix should be remover.

I thought i had fixed this before... Sorry..

Reply:
No problem. You can double check my final changes. I removed `MAGMA_LIBS` replacing with `HYPRE_MAGMA_LIB` and `HYPRE_MAGMA_LIB_DIR` where appropriate.